canonical-web-and-design/snapcraft.io
# Add snaps section

## Done
Added snaps view to the brand store

## QA
- Go to https://snapcraft-io-3700.demos.haus/admin/ahnuP3quahti9vis8aiw/snaps
- Check that the checkboxes on the table work as expected (e.g. header one selects all, unchecking one un-checks header etc.)
- Click "Include snap" and use the search to find and add snaps
- Submit the form and check the snaps are in the table
- Checkboxes and remove snaps
- Check that it works as expected

## Issue
Fixes #3652, #3653
